scripts/release_scripts/run_macaron.sh

```diff
@@ -439,6 +443,17 @@ if [[ -n "${arg_prov_exp:-}" ]]; then
     fi
 fi
 
+# Determine the provenance expectation path to be mounted into ${MACARON_WORKSPACE}/prov_files/${pf_name} where pf_name is a file name.
+if [[ -n "${arg_prov_file:-}" ]]; then
+    prov_file_path="${arg_prov_file}"
+    assert_path_exists "${prov_file_path}" "-pf/--provenance-expectation"
```

Comment:
This `assert_path_exists` check is not necessary. It exists only in the `--provenance_expectation` case because that argument expects either a file or a directory.
In this case of `--provenance-file`, it is always a file, which means we should use the `mount_file` function. This function already encapsulates path checking, so there is no need to do the `assert_path_exists` here.
```suggestion
```

Reply:
Removed in https://github.com/oracle/macaron/pull/698/commits/74f7bcf00de4f0f7eba0ae18de3a4be2790102a0